src/site/content/en/fast/image-cdns/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: Use image CDNs to optimize images
+authors:
+  - katiehempenius
+description: |
+  Image CDNs are excellent at optimizing images. 
+  Switching to an image CDN can yield a 40-80% savings in image bytes.
+date: 2019-07-28
+codelabs:
+  - codelab-cloudinary-url-api
+---
+
+## Why use an image CDN?
+
+Image CDNs are excellent at optimizing images. Switching to an image CDN can yield a [40-80% savings](https://www.youtube.com/watch?v=YJGCZCaIZkQ&t=1010s) in image bytes. In theory, it is possible to achieve the same results by using only build scripts - but in practice this is rare.
+
+
+## What is an image CDN?
+
+Image CDNs specialize in the transformation, optimization, and delivery of images. You can also think of them as an API for accessing and manipulating the images used on your site. For images loaded from an image CDN, an image URL indicates not only which image to load, but also parameters like size, format, and quality. This makes it easy to create different variations of an image for different use cases.
+
+
+
+<figure class="w-figure  w-figure--center">
+  <img src="./image-cdn-requests.jpg" alt="Shows the requests/responses to/from an image CDN">
```

Comment:
```suggestion
  <img class="w-screenshot" src="./image-cdn-requests.jpg" alt="Shows the requests/responses to/from an image CDN">
```

Reply:
Suggest adding the screenshot class here to give the image a border.

Reply:
also might be worth rethinking the `alt` text as it doesn't really describe the content of the image in a useful way. Might be worth trying something like "Shows the request/response flow between the image CDN and the client. Parameters like size and format are used to request variations of the same image."

Reply:
I also think it'd be worth adding a figcaption here, FWIW